app/code/community/Metrilo/Analytics/Model/Observer.php

```diff
@@ -278,19 +279,21 @@ private function _addToCart($productId, $item, $qty)
 
         $data =  array(
             'id'            => (int)$product->getId(),
+            'sku'           => $item->toArray()['sku'],
```

Comment:
`$item->getSku();` ?

Reply:
When I do that, I get the SKU of the child product instead of the SKU of the parent product. Fuck logic.